src/swim/swim.c

```diff
@@ -876,7 +873,22 @@ swim_parse_message(struct swim_context *ctx, swim_id_t from,
 			}
 		}
 	} else { /* iping request or response */
-		if (to != ctx->sc_target) {
+		if (to != ctx->sc_target &&
+		    upds[0].smu_state.sms_status == SWIM_MEMBER_SUSPECT) {
+			/* send dping request to iping target */
+			id_target = to;
+			id_sendto = to;
+			send_updates = true;
+
+			/* looking if sent already */
+			TAILQ_FOREACH(item, &ctx->sc_ipings, si_link) {
+				if (item->si_id == to) {
+					/* don't send the second times */
```

Comment:
*don't send a second time

Is any elaboration needed here?   Is there a timeout before they do send a second time?

Reply:
No. I just don't forward the second iping request to the same node which I already sent.